src/generatePythonFiles.sh

```diff
@@ -22,5 +22,11 @@ for path in [ # Search for libcircllhist.so
      pass
 if not C:
    # let dlopen throw it's error
+   print("""
```

Comment:
Should rethrow combined exception instead of printing to console here

Reply:
```python
if not C:
    try:
        ffi.dlopen("libcircllhist.so")
    except Exception as e:
        raise Exception(str(e) + """

libcircllhist.so was not found on your system.
Please install libcircllhist from: https://github.com/circonus-labs/libcircllhist/

        """)
```

Reply:
We effectively do that in the next line.